integration-tests/test/predeploys.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { expect } from 'chai'
+import { ethers } from 'hardhat'
+
+/* Imports: External */
+import { Contract, Wallet } from 'ethers'
+import { OptimismEnv } from './shared/env'
+import { getContractInterface } from '@eth-optimism/contracts'
+
+describe('ECDSAContractAccount', () => {
+  let l2Wallet: Wallet
+
+  const DEFAULT_TRANSACTION = {
```

Comment:
Perhaps its time to place `DEFAULT_TRANSACTION` in a helper file as its also defined here: https://github.com/ethereum-optimism/optimism/blob/c2b6e14bc470d71957928d51937293404d5e1a25/integration-tests/test/rpc.spec.ts#L25